### PR TITLE
EWL-3820 Topics | SG | Visual Requirements: "Related Content" Pattern

### PR DESCRIPTION
**Pull request by deadarm**
## Ticket(s)

**Jira Ticket**

- [EWL-3820: Topics | SG | Visual Requirements: "Related Content" Pattern](https://issues.ama-assn.org/browse/EWL-3820)

## Description

Styles the related content patterns.

## To Test

- [x] Navigate to Molecules > Lists >Topic Related Content list
- [x] Verify that the Topic Related Content list molecule includes the fields outlined in the requirements
- [x] Navigate to Molecules > Lists >Topic Related Content list with Images
- [x] Verify that the Topic Related Content list with Images molecule is styled to the specifications
- [x] Navigate to Organisms > Topic > Topic Related Content
- [x] Verify that the Topic Related Content organism is styled to the specifications
- [x] Navigate to Organisms > Topic > Topic Related Content with Images
- [x] Verify that the Topic Related Content with Images organism is styled to the specifications
- [x] Navigate to Templates > Topic > Topic
- [x] Verify that the Topic Related Content organism displays in the right rail and is styled to the specifications

## Relevant Screenshots/GIFs

screen shot 2017-08-04 at 2 16 22 pm

## Remaining Tasks

N/A

## Additional Notes
N/A

### CODE REVIEW

File: styleguide/source/_patterns/01-molecules/08-lists/13-list-topic-related-content/list-topic-related-content.twig [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ul class="list_topic-related-content">
+  <li class="list_item-topic-related-content">{% include 'atoms-h3' with { 'content': listTopicRelatedContent.first.title, 'class': 'list_topic-related-content_title' } %}{% include 'atoms-link-blue' with { 'content': 'Sed nuc', 'class': 'list_topic-related-content_link' } %}</li>
```
Comment (chelsiejohnston) [inline styleguide/source/_patterns/01-molecules/08-lists/13-list-topic-related-content/list-topic-related-content.twig]: Please adjust font size - article titles should be 24px size font and are showing up as 30px currently. Also, UX would like a `<h2>` for all article titles where possible, and an `<h3>` is included here

File: styleguide/source/_patterns/02-organisms/06-topic/04-topic-related-content/topic-related-content.twig [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div class="topic_related-content">
+  {% include 'atoms-h5' with { 'content': 'Lorem ipsum', 'class': 'topic_related-content_title' } %}
```
Comment (chelsiejohnston) [inline styleguide/source/_patterns/02-organisms/06-topic/04-topic-related-content/topic-related-content.twig]: Suggestion to use existing atom `atoms-title-label` here. UX would like an `<h3>` for the element, and we're already using it elsewhere in the template

File: styleguide/source/_patterns/03-templates/04-topic/_topic.scss [hunk 3]
```diff
@@ -22,3 +22,12 @@
     order: 3;
   }
 }
+
+.topic_rail-right {
+  div {
+    margin-top: $gutter;
```
Comment (chelsiejohnston) [inline styleguide/source/_patterns/03-templates/04-topic/_topic.scss]: May need to be changed awaiting discussion in #171

File: styleguide/source/_patterns/02-organisms/06-topic/04-topic-related-content/topic-related-content.twig (hunk 2 above)
Comment (chelsiejohnston) [inline styleguide/source/_patterns/02-organisms/06-topic/04-topic-related-content/topic-related-content.twig]: There should also be a 1px solid gray border/`<hr>` underneath the Topic Label element (you can see it in the wires). Sorry if I missed this in the visual reqs!

File: styleguide/source/_patterns/03-templates/04-topic/_topic.scss (hunk 3 above)
Comment (deadarm) [inline styleguide/source/_patterns/03-templates/04-topic/_topic.scss]: Removing this.

**Pull request merged**